### PR TITLE
Stabilize Blur & Custom Background

### PR DESCRIPTION
**Pull request by prabhjot-msft**
# What
Stabilize Blur & Custom Background

# Why

# How Tested

# Process & policy checklist

- [ ] I have updated the project documentation to reflect my changes if necessary.
- [ ] I have read the [CONTRIBUTING](https://github.com/Azure/communication-ui-library/blob/main/CONTRIBUTING.md) documentation.

**Is this a breaking change?**

- [ ] This change causes current functionality to break.

### CODE REVIEW

File: packages/react-composites/review/stable/react-composites.api.md [hunk 1]
```diff
@@ -160,10 +169,18 @@ export interface CallAdapterCallOperations {
     startCamera(options?: VideoStreamOptions): Promise<void>;
     startCaptions(options?: StartCaptionsOptions): Promise<void>;
     startScreenShare(): Promise<void>;
+    // @beta
```
Comment (mgamis-msft) [inline packages/react-composites/review/stable/react-composites.api.md:172]: These new properties should not have the beta annotation.

Comment (prabhjot-msft) [reply packages/react-composites/review/stable/react-composites.api.md]: Done

File: packages/calling-stateful-client/review/stable/calling-stateful-client.api.md [hunk 2]
```diff
@@ -83,7 +85,7 @@ export type CallErrors = {
...
+export type CallErrorTarget = 'Call.addParticipant' | 'Call.dispose' | 'Call.feature' | 'Call.hangUp' | 'Call.hold' | 'Call.mute' | 'Call.muteIncomingAudio' | 'Call.off' | 'Call.on' | 'Call.removeParticipant' | 'Call.resume' | 'Call.sendDtmf' | 'Call.startAudio' | 'Call.startScreenSharing' | 'Call.startVideo' | 'Call.stopScreenSharing' | 'Call.stopAudio' | 'Call.stopVideo' | 'Call.unmute' | 'Call.unmuteIncomingAudio' | 'CallAgent.dispose' | 'CallAgent.feature' | 'CallAgent.join' | 'CallAgent.off' | 'CallAgent.on' | 'CallAgent.startCall' | 'CallClient.createCallAgent' | 'CallClient.createTeamsCallAgent' | 'CallClient.feature' | 'CallClient.getDeviceManager' | 'DeviceManager.askDevicePermission' | 'DeviceManager.getCameras' | 'DeviceManager.getMicrophones' | 'DeviceManager.getSpeakers' | 'DeviceManager.off' | 'DeviceManager.on' | 'DeviceManager.selectMicrophone' | 'DeviceManager.selectSpeaker' | 'IncomingCall.accept' | 'IncomingCall.reject' | /* @conditional-compile-remove(video-background-effects) */ 'VideoEffectsFeature.startEffects';
```
Comment (mgamis-msft) [inline packages/calling-stateful-client/review/stable/calling-stateful-client.api.md:88]: I don't think /* @conditional-compile-remove(video-background-effects) */ should be in the stable api.md. I could be wrong though.

File: change-beta/@azure-communication-react-6ebfab2b-bdaa-48d1-ba30-b371de8eea75.json [hunk 3]
```diff
@@ -0,0 +1,9 @@
+{
+  "type": "minor",
+  "area": "feature",
+  "workstream": "Calling",
+  "comment": "stabilize blur",
```
Comment (JamesBurnside) [inline change-beta/@azure-communication-react-6ebfab2b-bdaa-48d1-ba30-b371de8eea75.json:5]: This is what will be in the changelog - we want to celebrate a new feature going to GA! `stabilize blur` isn't indicative that  this is a new GA feature.
Unless there's a plan to update this elsewhere, perhaps something like: `General Availability release of Background Effects - your users can now blur or replace their video background`

File: packages/calling-stateful-client/review/stable/calling-stateful-client.api.md (hunk 2 above)
Comment (JamesBurnside) [reply packages/calling-stateful-client/review/stable/calling-stateful-client.api.md:88]: this annoying gets leaked in as an artifact of api-extractor, it won't impact the build or the devx experience but for these might be worth manually deleteing the conditional compile lines so they don't appear in the api.md files

File: packages/react-components/review/stable/react-components.api.md [hunk 4]
```diff
@@ -97,6 +97,7 @@ export interface CameraButtonProps extends ControlBarButtonProps {
     enableDeviceSelectionMenu?: boolean;
     localVideoViewOptions?: VideoStreamOptions;
     onSelectCamera?: (device: OptionsDevice) => Promise<void>;
+    onShowVideoEffectsPicker?: (showVideoEffectsOptions: boolean) => void;
```
Comment (JamesBurnside) [inline packages/react-components/review/stable/react-components.api.md:100]: nit: this should probably be onVideoEffectsClick and the implementation to show a picker onclick is not tied to the button but is composite behavior

Comment (JamesBurnside) [reply packages/react-components/review/stable/react-components.api.md:100]: What does the argument `showVideoEffectsOptions` do again? (why is Options a boolean?)

**Review by JamesBurnside — APPROVED** [review]

File: packages/react-components/review/stable/react-components.api.md (hunk 4 above)
Comment (prabhjot-msft) [reply packages/react-components/review/stable/react-components.api.md]: Changed the name to `onClickVideoEffects` and this boolean is used to show/hide the picker changed the name to `showVideoEffects`

File: packages/calling-stateful-client/review/stable/calling-stateful-client.api.md (hunk 2 above)
Comment (mgamis-msft) [reply packages/calling-stateful-client/review/stable/calling-stateful-client.api.md]: I see. That's fine then if it does not affect the build.

**Review by mgamis-msft — APPROVED** [review]

**Pull request merged**